mlir/include/mlir/Conversion/Passes.td

```diff
@@ -780,6 +780,22 @@ def ConvertMathToSPIRV : Pass<"convert-math-to-spirv"> {
   let dependentDialects = ["spirv::SPIRVDialect"];
 }
 
+//===----------------------------------------------------------------------===//
+// MathToEmitC
+//===----------------------------------------------------------------------===//
+
+def ConvertMathToEmitC  : Pass<"convert-math-to-emitc", "ModuleOp"> {
+  let summary = "Convert some Math operations to EmitC Call_opaque";
+  let description = [{
+    This pass converts supported Math ops to opaque_call ops targeting libc/libm functions.
+    Unlike convert-math-to-funcs pass, this pass uses call_opaque,
+    therefore enables us to overload the same funtion with different argument types
+  }];
+  let dependentDialects = ["emitc::EmitCDialect",
+                           "math::MathDialect"
+  ];
```

Comment:
```suggestion
  let dependentDialects = ["emitc::EmitCDialect"];
```
If I am correct you only need to add the dialect here from that you produce ops not the one you start from.

Reply:
You were right. I deleted "math::MathDialect"